src/GSDBroadenPeak.ts

```diff
@@ -1,7 +1,11 @@
+import { Shape1D } from 'ml-peak-shape-generator';
+
 export interface GSDBroadenPeak {
+  id?: string;
   x: number;
   y: number;
   width: number;
+  shape?: Shape1D;
```

Comment:
Do we really know the shape ? I think I didn't put it because it was not really defined.

Reply:
BroadenPeak increases the width of the peak by a factor, the input is a `GSDPeak` structure but the property `shape` was added, that is why I added shape as optional in `GSDBroadenPeak`

Reply:
ok